docs/quick-tutorial.md

```diff
@@ -1,48 +1,48 @@
+Thanos bases itself on vanilla [Prometheus](https://prometheus.io/) (v2.2.1+). We plan to support *all* Prometheus versions beyond this one.
 
-NOTE: It is highly recommended to use Prometheus v2.13+ due to Prometheus remote read improvements.
+NOTE: It is highly recommended to use Prometheus v2.13.0+ due to its remote read improvements.
 
-Always make sure to run Prometheus as recommended by Prometheus team, so:
+Always make sure to run Prometheus as recommended by the Prometheus team:
 
-* Put Prometheus in the same failure domain. This means same network, same datacenter as monitoring services.
-* Use persistent disk to persist data across Prometheus restarts.
+* Put Prometheus in the same failure domain. This means in the same network and in the same geographic location as the monitored services.
+* Use a persistent disk to persist data across Prometheus restarts.
 * Use local compaction for longer retentions.
-* Do not change min TSDB block durations.
-* Do not scale out Prometheus unless necessary. Single Prometheus is highly efficient (:
+* Do not change the minimum TSDB block durations.
+* Do not scale out Prometheus unless necessary. A single Prometheus instance is already efficient.
 
 We recommend using Thanos when you need to scale out your Prometheus instance.
 
 ## Components
 
-Following the [KISS](https://en.wikipedia.org/wiki/KISS_principle) and Unix philosophies, Thanos is made of a set of components with each filling a specific role.
+Following the [KISS](https://en.wikipedia.org/wiki/KISS_principle) and Unix philosophies, Thanos is comprised of a set of components where each fulfills a specific role.
 
 * Sidecar: connects to Prometheus, reads its data for query and/or uploads it to cloud storage.
 * Store Gateway: serves metrics inside of a cloud storage bucket.
-* Compactor: compacts, downsamples and applies retention on the data stored in cloud storage bucket.
-* Receiver: receives data from Prometheus's remote-write WAL, exposes it and/or upload it to cloud storage.
-* Ruler/Rule: evaluates recording and alerting rules against data in Thanos for exposition and/or upload.
+* Compactor: compacts, downsamples and applies retention on the data stored in the cloud storage bucket.
+* Receiver: receives data from Prometheus's remote write write-ahead log, exposes it, and/or uploads it to cloud storage.
+* Ruler/Rule: evaluates recording and alerting rules against data in Thanos for exposure and/or upload.
 * Querier/Query: implements Prometheus's v1 API to aggregate data from the underlying components.
-* Query Frontend: implements Prometheus's v1 API proxies it to Query while caching the response and optional splitting by queries day.
+* Query Frontend: implements Prometheus's v1 API to proxy it to Querier while caching the response and optionally splitting it by queries per day.
 
-Deployment with Sidecar for Kubernetes:
+Deployment via Kubernetes sidecar:
```

Comment:
Sidecar is another Thanos component and should be left capitalized
```suggestion
Deployment with Thanos Sidecar for Kubernetes:
```

Reply:
Hmmm, this is an interesting case. I could see keeping it capitalized as an official component of Thanos. But on the other hand, I'm not clear how it differs from the standard concept of a Kubernetes sidecar. It's sort of a judgement call; you guys probably know best so I'm okay to go that route if this way sounds more acceptable to the Thanos community. We'll also want to bring https://github.com/thanos-io/thanos/blob/main/docs/components/sidecar.md into parity soon too then.

But if we go the capitalization route, then we'll generally need to drop `a` and `the` before it, as it will become a proper noun, so some of the other suggestions will need to be revised accordingly.